pkg/manager/member/tidb_member_manager.go

```diff
@@ -566,6 +568,32 @@ func getNewTiDBSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		})
 	}
 
+	var additionalVolumeClaims []corev1.PersistentVolumeClaim
+	if len(tc.Spec.TiDB.StorageVolumes) > 0 {
+		for _, storageVolume := range tc.Spec.TiKV.StorageVolumes {
+			quantity, err := resource.ParseQuantity(storageVolume.StorageSize)
+			if err != nil {
+				klog.Errorf("Cannot parse storage size %v in Spec.TiDB.StorageVolumes, tidbcluster %s/%s, error: %v", storageVolume.StorageSize, tc.Namespace, tc.Name, err)
+				continue
+			}
+			storageRequest := corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: quantity,
+				},
+			}
+			var storageClassName *string
+			if storageVolume.StorageClassName != nil && len(*storageVolume.StorageClassName) > 0 {
+				storageClassName = storageVolume.StorageClassName
+			} else {
+				storageClassName = tc.Spec.TiKV.StorageClassName
```

Comment:
TiKV ?

Reply:
e.. sorry, this is a proposal I temporarily submitted. I will  optimize this pr.